components/script/dom/windowproxy.rs

```diff
@@ -393,9 +488,15 @@ impl WindowProxy {
             "" => DOMString::from("_blank"),
             _ => target,
         };
-        // TODO Step 4, properly tokenize features.
+        // Step 4
+        let features = WindowProxy::tokenize_open_features(features);
         // Step 5
-        let noopener = features.contains("noopener");
+        let noreferrer = WindowProxy::parse_open_feature_boolean(&features, "noreferrer");
```

Comment:
I would say that the `parse_` functions and `tokenize_open_features` could simply be free functions in the current module, as opposed to methods on `WindowProxy`.